benchmark.go

```diff
@@ -0,0 +1,435 @@
+package timber
```

Comment:
I wouldn't put this in the public api of the timber package. maybe make a benchmark subpackage.